examples/v1beta1/taskruns/alpha/param_array_indexing.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  name: array-with-default-1
+spec:
+  params:
+    - name: array-to-echo
+      value:
+        - "foo"
+        - "bar"
+  taskSpec:
+    params:
+      - name: array-to-echo
+        type: array
+    steps:
+      # this step should echo "foo"
+      - name: echo-params
+        image: bash:3.2
+        args: [
+          "echo",
+          "$(params.array-to-echo[0])",
+        ]
```

Comment:
Can you add another step in this to test `args: ["echo $(params.array-to-echo[0])"]`? or `args: ["echo $(params.array-to-echo[0])"] > $(workspaces.testworkspace.path)/testfile.txt` something like that?

Reply:
this seems not the correct syntax, did you mean
`args: ["echo $(params.array-to-echo[0])"]`?

Reply:
Ah yes. I saw you added that. You can close this thread now